pennylane_lightning/core/src/simulators/lightning_kokkos/gates/MatrixGateFunctors.hpp

```diff
@@ -60,7 +60,7 @@ template <class Precision> struct multiQubitOpFunctor {
                         const KokkosComplexVector &matrix_,
                         const std::vector<std::size_t> &wires_) {
         wires = vector2view(wires_);
-        dim = exp2(wires_.size());
+        dim = Pennylane::Util::exp2(wires_.size());
```

Comment:
We could also bring this the scope at the top of the file similar to [here](https://github.com/PennyLaneAI/pennylane-lightning/blob/9eb61230724131d3fca036a09d1e84771aafacfa/pennylane_lightning/core/src/simulators/lightning_qubit/StateVectorLQubitManaged.hpp#L39) for a slightly better code quality. What do you think?

Reply:
that will make it a lot cleaner. I'll update it, thanks!

Reply:
i've now tried to change it, but hipcc really struggles to resolve the correct exp2 even when putting `using Pennylane::Util::exp2;` at the start... so I will put the namespaces back